internal/db/models/task.go

```diff
@@ -0,0 +1,72 @@
+package models
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"gorm.io/gorm"
+)
+
+type TaskStatus string
+
+const (
+	TaskStatusUnknown   TaskStatus = "unknown"
+	TaskStatusPending   TaskStatus = "pending"
+	TaskStatusRunning   TaskStatus = "running"
+	TaskStatusCompleted TaskStatus = "completed"
+	TaskStatusFailed    TaskStatus = "failed"
+)
+
+type Task struct {
```

Comment:
I think this should have OwnerID as well so we can track who originated the task. 

In most cases it should be the same as the project's ownerID, but thinking about the testnets, we might have multiple users that want to execute tasks on a project.

Reply:
we need to discuss it as someone might need to play with someone else's task within the same project.

Reply:
added a note to #153 to include this as well.